### PR TITLE
docs: added FAQs and Tips page

### PR DESCRIPTION
**Pull request by andersonaddo**
"Fixes" #3508

So here's a look at what I was thinking about. Unfortunately, I only have one thing to add to it at the time, but hopefully it'll grow over time 😛

Also, do you think it may be smart to move over the things in [Miscellaneous](https://rnfirebase.io/#miscellaneous) over to this page?

### CODE REVIEW

File: docs/faqs-and-tips.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+title: FAQs and Tips
+description: Learn about top tips from and common problems faced by the React Native Firebase community.
+next: /releases
+previous: /migrating-to-v6
```
Comment (Ehesp) [inline docs/faqs-and-tips.md:5]: The `/releases` pages also needs the `previous` tag changing.

**Comment by Ehesp** [comment]
Thanks for this - I'll get it reviewed and think of some other topics which can be added.

File: docs/faqs-and-tips.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+---
+
+Over the years, there’s been a lot of discussions on our [Github]( https://github.com/invertase/react-native-firebase) and [Discord]( https://invertase.link/discord). Many of them have been about common problems developers face when using our package, and some of them resulted in very good advice being given.
```
Comment (Salakar) [inline docs/faqs-and-tips.md:8]:
```suggestion
Over the years, there’s been a lot of discussions on our [GitHub]( https://github.com/invertase/react-native-firebase) and [Discord]( https://invertase.link/discord). Many of them have been about common problems developers face when using our package, and some of them resulted in very good advice being given.
```

File: docs/faqs-and-tips.md (hunk 1 above)
Comment (andersonaddo) [reply docs/faqs-and-tips.md:5]: Done!

File: docs/faqs-and-tips.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+
+# FAQs
+
+Here are some commonly encountered problems:
+
+[TODO]
+
+# Tips
+ - Advice on cleanly creating a test and production environment for your React Native Firebase App: [#3504]( https://github.com/invertase/react-native-firebase/issues/3504)
```
Comment (mikehardy) [inline docs/faqs-and-tips.md:21]:
```suggestion
 - Advice on supporting multiple environments (for example, dev, prod, maybe also staging, qa) for your React Native Firebase App: [#3504]( https://github.com/invertase/react-native-firebase/issues/3504)
```

Comment (mikehardy) [reply docs/faqs-and-tips.md:21]: Meant to provide some text as well - I love this initiative, and just for that specific one I wanted to give more context leading up to the link. But just getting it going is great.

Another FAQ that always comes up @Ehesp / @Salakar are the classes of errors that lead to the answer 'run `npx react-native-clean-project && cd ios && pod install` - like differences between debug and release on iOS, version mismatch of packages

**Comment by andersonaddo** [comment]
👀

**Comment by andersonaddo** [comment]
Just bumping this so it doesn't get forgotten :)

File: docs/faqs-and-tips.md (hunk 1 above)
Comment (Salakar) [reply docs/faqs-and-tips.md:5]: Thanks

**Comment by andersonaddo** [comment]
-ᄒᴥᄒ-

**Comment by Salakar** [comment]
Not forgotten about this sorry, still waiting on @ehesp to review

**Comment by Salakar** [comment]
Lets just ship this, looks ok to me, merge conflicts would need to be updated of course and we an just remove the `FAQs` heading & section until we have some faqs, or add in the v5 (#3791) one there to get it going

**Comment by andersonaddo** [comment]
I think I've resolved everything!

**Comment by Salakar** [comment]
Nice, thanks. Shipped! Should appear on the site shortly

**Comment by Salakar** [comment]
https://rnfirebase.io/faqs-and-tips

giff

**Pull request merged**